snakemake.el

```diff
@@ -351,6 +356,46 @@ targets."
              " "))
 
 
+;;; Graphing commands
+
+(defun snakemake-show-dag ()
+  "Create and show dag in emacs-buffer."
+  (interactive)
+  (find-file (snakemake-create-dag)))
+
+(defun snakemake-show-rulegraph ()
+  "Create and show rulegraph in emacs-buffer."
+  (interactive)
+  (find-file (snakemake-create-rulegraph)))
+
+(defun snakemake-create-dag ()
+  "Create dag for workflow.
+
+  Shows all jobs. If the graph becomes too crowded, use the rulegraph function."
```

Comment:
Elisp docstrings don't indent the second line.  This renders as

    Create dag for workflow.

      Shows all [...]

if in the help buffer.